grails-app/controllers/de/iteratec/osm/result/EventResultDashboardController.groovy

```diff
@@ -352,6 +375,24 @@ class EventResultDashboardController {
         modelToRender.put('i18n', i18n as JSON)
     }
 
+    /**
+     * Validates the command and creates an error message string if necessary.
+     * @param cmd
+     * @return a string containing the error messages in html format or an empty string if the command is valid
+     */
+    private String getErrorMessages(GetLinechartCommand cmd) {
```

Comment:
Grails [command objects](http://docs.grails.org/latest/guide/theWebLayer.html#commandObjects) are the correct artifact for parameter validation. GetLinechartCommand implements interface Validateable. So you should use its features instead of re-implementing validation on your own here.